website/payments/models.py

```diff
@@ -179,6 +179,27 @@ class Payment(models.Model):
         null=True,
     )
 
+    moneybird_invoice_id = models.CharField(
+        verbose_name=_("moneybird invoice id"),
+        max_length=255,
+        blank=True,
+        null=True,
+    )
+
+    moneybird_financial_statement_id = models.CharField(
+        verbose_name=_("moneybird financial statement id"),
+        max_length=255,
+        blank=True,
+        null=True,
+    )
+
+    moneybird_financial_mutation_id = models.CharField(
+        verbose_name=_("moneybird financial mutation id"),
+        max_length=255,
+        blank=True,
+        null=True,
+    )
+
```

Comment:
This should, similar to the moneybird contact, be a separate model in the moneybird app with a one-to-one key to payments. We want to separate payments from moneybird sync, so even when moneybird breaks, payments doesn't break.